tests/common/test_transaction.py

```diff
@@ -482,6 +483,26 @@ def test_add_output_to_tx_with_invalid_parameters(asset_definition):
         tx.add_output('somewronginput')
 
 
+def test_sign_single_fulfillment_outputs_correct_uri(utx, user_priv):
+    from cryptoconditions import Ed25519Fulfillment
+    from cryptoconditions.crypto import Ed25519SigningKey
+    utx_dict = utx.to_dict()
+    utx_dict['inputs'][0]['fulfillment'] = None
+    message = rapidjson.dumps(
+        utx_dict,
+        skipkeys=False,
+        ensure_ascii=False,
+        sort_keys=True,
+    ).encode()
```

Comment:
Just as a piece of information, for the purpose of testing, `json` is sufficient, i.e., `rapidjson` is not essential, except for its better performance.